liquibase-core/src/test/groovy/liquibase/resource/ClassLoaderResourceAccessorTest.groovy

```diff
@@ -1,104 +1,105 @@
 package liquibase.resource
 
-
+import liquibase.test.TestContext
 import liquibase.util.StreamUtil
 import spock.lang.Specification
 import spock.lang.Unroll
 
-import java.nio.file.FileSystem
-
 class ClassLoaderResourceAccessorTest extends Specification {
 
     def testResourceAccessor = new ClassLoaderResourceAccessor(new URLClassLoader(
             [
                     Thread.currentThread().getContextClassLoader().getResource("simple-files.jar"),
                     Thread.currentThread().getContextClassLoader().getResource("simple-files.zip"),
             ] as URL[],
-            Thread.currentThread().getContextClassLoader()))
+            new URLClassLoader([
+                    new File(TestContext.getInstance().findCoreJvmProjectRoot(), "/target/classes").toURL(),
+                    new File(TestContext.getInstance().findCoreJvmProjectRoot(), "/target/test-classes").toURL(),
+            ] as URL[]
+            )
+    ))
+
+//    @Unroll("#featureName: #relativeTo #streamPath")
+//    def "getFinalPath"() {
+//        expect:
+//        new ClassLoaderResourceAccessor().getFinalPath(relativeTo, streamPath) == expected
+//
+//        where:
+//        relativeTo                         | streamPath                       | expected
+//        null                               | "com/example/test.sql"           | "com/example/test.sql"
+//        null                               | "/com/example/test.sql"          | "com/example/test.sql"
+//        null                               | "\\com\\example\\test.sql"       | "com/example/test.sql"
+//        null                               | "/com////example//test.sql"      | "com/example/test.sql"
+//        null                               | "classpath:com/example/test.sql" | "com/example/test.sql"
+//        "com/example"                      | "test.sql"                       | "com/example/test.sql"
+//        "/com/example/"                    | "test.sql"                       | "com/example/test.sql"
+//        "com/example"                      | "/my/test.sql"                   | "com/example/my/test.sql"
+//        "com/example"                      | "/my/test.sql"                   | "com/example/my/test.sql"
+//        "com/example/other.file"           | "/my/test.sql"                   | "com/example/my/test.sql"
+//        "classpath:com/example/other.file" | "my/test.sql"                    | "com/example/my/test.sql"
+//        "changelog.xml"                    | "sql/function.sql"               | "sql/function.sql"
+//        "db-change.log/changelog.xml"      | "data/file.csv"                  | "db-change.log/data/file.csv"
+//    }
```

Comment:
No, it's for a method that was removed so I should complete delete the tests. I had left them there in case I needed them for anything, but it ended up not.